TTHAnalysis/python/plotter/susy-sos/pack_fake_rates_data.py

```diff
@@ -0,0 +1,207 @@
+from math import *
```

Comment:
it would be better to refactor the fake rate scripts, avoiding code duplication with the same ones used for ttH and susy-multilepton